subsys/sal/sid_pal/src/sid_crypto.c

```diff
@@ -22,6 +22,41 @@
 
 LOG_MODULE_REGISTER(sid_crypto, CONFIG_SIDEWALK_CRYPTO_LOG_LEVEL);
 
+#ifdef CONFIG_SIDEWALK_CRYPTO_PERF_TEST
```

Comment:
I'm not feeling comforatable with adding a special code to this module just for purpose of the one test.
If we need to measure Sidewalk performance, we should opearte on Sidewalk (sid_pal) api in tests.
If we need to measure ncs crypto (psa api), we should create a test in ncs.
But it's just a comment to open discussion, not necessary a blocker for merge.